doc/cmake/INSTALL.md

```diff
@@ -35,10 +38,11 @@ The general commands to download, configure and install WRF-CMake on Linux and m
 git clone https://github.com/WRF-CMake/wrf.git
 cd wrf
 mkdir build && cd build
-cmake -DCMAKE_INSTALL_PREFIX=<install_directory> ..
-make install
+cmake -GNinja -DCMAKE_INSTALL_PREFIX=<install_directory> ..
+cmake --build .
+cmake --install .
 ```
-where `<install_directory>` is the directory where to install WRF. Depending on your system's configuration, you may need to specify [WRF-CMake options](#wrf-cmake-options). If multiple compilers are available on the system, use the `CC` (C compiler) and/or `FC` (Fortran compiler) environment variables to specify them. For example, to use Intel C and Fortran compilers run `CC=icc FC=ifort cmake -DCMAKE_INSTALL_PREFIX=<install_directory> ..`. On macOS, use `CC=gcc-8 FC=gfortran-8` to use the GNU compilers installed with Homebrew. If your system has enough memory you can enable parallel compilation with `make install -j <n>` where `<n>` is the maximum number of jobs you like to run in parallel.
+where `<install_directory>` is the directory where to install WRF. Depending on your system's configuration, you may need to specify [WRF-CMake options](#wrf-cmake-options). If multiple compilers are available on the system, use the `CC` (C compiler) and/or `FC` (Fortran compiler) environment variables to specify them. For example, to use Intel C and Fortran compilers run `CC=icc FC=ifort cmake -GNinja -DCMAKE_INSTALL_PREFIX=<install_directory> ..`. On macOS, use `CC=gcc-8 FC=gfortran-8` to use the GNU compilers installed with Homebrew. If your system has enough memory you can enable parallel compilation with `cmake --build . --j <n>` where `<n>` is the maximum number of jobs you like to run in parallel.
```

Comment:
I think it's `-- -j 1` or since CMake 3.12 just `-j 1`, see `https://stackoverflow.com/a/50883540.